.github/workflows/async-auto-label.yml

```diff
@@ -2,7 +2,10 @@
 name: Apply requested async label
 
 'on':
-  workflow_call:
+  issue_comment:
+    types:
+      - created
+      - edited
```

Comment:
This makes sense for the workflow in O2/O2Physics/etc, but in ali-bot, it should stay as `workflow_call`. Then one shared workflow in ali-bot can be called and reused from all the other repos, so we don't have to copy-paste the same file everywhere.

Reply:
I see. So it means we leave `workflow_call` here and it would then be made sure to have
```yaml
issue_comment:
    types:
      - created
      - edited
```
 in some implementation where this is called, right?

Reply:
Yes, exactly! For an example of the mechanism, see e.g. `.github/workflows/pr-security-approval.yml` in ali-bot, alidist and O2Physics.